leaks.go

```diff
@@ -56,14 +56,13 @@ func Find(options ...Option) error {
 
 	opts := buildOpts(options...)
 	var stacks []stack.Stack
-	retry := true
-	for i := 0; retry; i++ {
+	retry := opts.newRetry()
+	for retry() {
```

Comment:
The `retry` variable can be scoped to the `for` loop,
```suggestion
	for retry := opts.newRetry(); retry(); {
```